### DISCUSSION TITLE
Can I increase the number of gradient stops?

### DISCUSSION THREAD

**Discussion by nehiro** [tailwindlabs/tailwindcss#11799] [Help] [answered]
Hi. Can I currently add more than two via-{color} in a gradient?

https://tailwindcss.com/docs/gradient-color-stops

**Comment by wongjn** [comment, marked as answer]
Nope! Consider having the gradient in a [custom background image](https://tailwindcss.com/docs/background-image#customizing-your-theme) or an [arbitrary value](https://tailwindcss.com/docs/background-image#arbitrary-values) instead.
